docs/pyproject.toml

```diff
@@ -12,7 +12,7 @@ mkdocs-material = "^8.3.7"
 mknotebooks = "^0.7.1"
 pyppeteer = "^1.0.2"
 nest-asyncio = "^1.5.5"
-dpdocsutils = {path = "dpdocsutils"}
+dpdocsutils = {path = "dpdocsutils", develop = true}
```

Comment:
Can `dpdocsutils` be a module rather than a package? I don't think it's used anywhere else or has different dependencies to the main docs project.

Reply:
I'm assuming it needs to be a separate package so it can be seen from the Notebooks?

@shahinrostami to confirm; however it's out of scope for this PR regardless

Reply:
@mands @antdking that's right, creating a package wasn't my first choice but I didn't want to overcomplicate the notebooks, e.g. https://stackoverflow.com/questions/34478398/import-local-function-from-a-module-housed-in-another-directory-with-relative-im

Reply:
Got it - i guess we can mess around with the call sites instead to modify the pythonpath rather than the notebooks, i.e. in `nbbuild.sh` but I don't think it's a priority or anything